targets/targets.json

```diff
@@ -8030,7 +8030,6 @@
     "ARM_FM": {
         "inherits": ["Target"],
         "public": false,
-        "macros": ["__ARM_FM"],
```

Comment:
please replace to 
```
"macros": ["SKIP_TIME_DRIFT_TESTS"],
```

Reply:
Ohh now I understand. Sorry I got a bit confused with these symbols.

Now in case of `ARM_FM` symbol `SKIP_TIME_DRIFT_TESTS` will be defined twice (here and in CI `greentea.groovy` script). Probably this is not a problem?

Reply:
fixed.

Reply:
Should not be a problem